lib/stale.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Determines if an issue or PR is stale and create a message if needed.
+ * Will only process when the configuration for days considered stale
+ * is set for Pulls and Issues respectively.
+ *
+ * Assumes that the numbers of days are always set if stale property in Settings
+ * is defined.
+ *
+ * @return
+ *  JSON object with the properties mergeable and the description if
+ *  `mergeable` is false
+ */
+module.exports = async (context, settings) => {
+  return process(context, settings)
+}
+
+const process = async (context, config) => {
```

Comment:
i notice that these functions doesn't return anything, i am curious about what the behavior would be if we `await process()`.